types/index.d.ts

```diff
@@ -607,7 +607,7 @@ declare module "jspdf" {
   export interface TableConfig {
     printHeaders?: boolean;
     autoSize?: boolean;
-    margins?: number;
+    margins?: : {top?: number, left?: number, width?: number, bottom?: number};
```

Comment:
Thanks for the PR. I think `width` should be `right`.

Reply:
checking the code i saw that it is also called `margins.width` and `margins.right`... but not sure if it is used 🤔
https://github.com/parallax/jsPDF/blob/2d9a91916471f1fbe465dbcdc05db0cf22d720ec/src/modules/cell.js#L41

https://github.com/parallax/jsPDF/blob/2d9a91916471f1fbe465dbcdc05db0cf22d720ec/src/modules/cell.js#L51-L52

https://github.com/parallax/jsPDF/blob/2d9a91916471f1fbe465dbcdc05db0cf22d720ec/src/modules/cell.js#L403-L405

Reply:
> Thanks for the PR. I think `width` should be `right`.

In the link I included in the PR it is actually with, not right:

![image](https://user-images.githubusercontent.com/63733780/223225738-b3c19484-4866-4558-a1c0-50491782a0e9.png)